src/compiler/moduleNameResolver.ts

```diff
@@ -816,6 +818,33 @@ namespace ts {
         return loadNodeModuleFromDirectory(extensions, candidate, failedLookupLocations, onlyRecordFailures, state, considerPackageJson);
     }
 
+    function parseNodeModuleFromPath(path: string): { packageDirectory: string, subModuleName: string } | undefined {
+        const nodeModules = "/node_modules/";

```

Comment:
Is this case-sensitive even on Windows?

Reply:
Should be fixed if we normalize the path first.